graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -79,7 +79,7 @@
   being renamed or suppressed.
 """
 from collections import namedtuple
-from typing import AbstractSet, Any, Dict, List, Mapping, Optional, Set, Tuple, Union, cast
+from typing import AbstractSet, Any, Dict, List, Optional, Protocol, Set, Tuple, Union, cast
```

Comment:
`Protocol` is new in 3.8: https://docs.python.org/3/library/typing.html#typing.Protocol

This is why the 3.6 and 3.7 builds are failing. Fortunately, `typing_extensions` has a backport available, so just add to the conditional import in the top-level `typedefs.py` file that I tweaked to add `Literal` earlier.